### ISSUE TITLE
[Windows]: Putting my customized `.esformatter` file at `~/` will not work while my code place in another disk like `D:/`

### ISSUE THREAD

**Issue by leftstick** [millermedeiros/esformatter#336]
I found that:

```
When running esformatter executable on the terminal, it will look for the closest .esformatter file and use that as a setting unless you specify --preset default or --config ./path/to/config_file.json.
```

For mac os and linux, one `.esformatter` file placed at `~/` will work for my work station.

But it doesn't work if i am using windows.

Is there any good solution for this issue?

Thanks

**Comment by millermedeiros** [comment]
@leftstick did you try with version 0.7.1?

after 0.7.1 we started using https://www.npmjs.com/package/user-home/ so you should put the file at the path that it returns.

please add a comment here telling which path is used on windows and/or update the documentation, I have no idea which path it uses and don't have a windows machine/vm to test...

**Comment by leftstick** [comment]
Thanks for quick reply. I tested on my windows env, and `user-home` just returned the same value as `~/`.

And the file `.esformatter` doesn't seem to work while placed in `~/`.

**Comment by leftstick** [comment]
Code `/lib/options.js` line 92 ~ 96 as following:

```javascript
  var rc = findAndMergeConfigs(basedir);
  if (!rc && basedir !== cwd) {
    rc = findAndMergeConfigs(cwd);
  }
  return merge(rc || getGlobalConfig(), customOptions);
```

`rc` is returned as `{}`, which means, `getGlobalConfig()` will never be called in the case.

**Comment by millermedeiros** [comment]
so it looks like the problem is that `findAndMergeConfigs` should return `null` if `!config && !parentConfig`. pull requests are highly appreciated, specially since you're able to test.

**Comment by millermedeiros** [comment]
just published v0.7.2 that includes this fix. thanks once again.
